openpgp/packet/signature.go

```diff
@@ -860,8 +935,7 @@ func (sig *Signature) buildSubpackets() (subpackets []outputSubpacket, err error
 		if err != nil {
 			return
 		}
-		subpackets = append(subpackets, outputSubpacket{true, embeddedSignatureSubpacket, true,
-			buf.Bytes()})
+		subpackets = append(subpackets, outputSubpacket{true, embeddedSignatureSubpacket, true, buf.Bytes()})
```

Comment:
Not that it matters too much, but I would try to limit the amount of trivial changes in this PR, because they make it more difficult to see which changes are important. Also for some whitespace changes above; I would just take a quick pass through the PR and remove everything that's not necessary.

Reply:
Maybe this was `go fmt`, not sure. But the above is unreadable to me (on the very least, break the line after `{`?)